app/main.py

```diff
@@ -1,13 +1,40 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"], "column": error["column_number"],
+        "message": error["text"], "name": error["code"], "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [
+            {
+                "line": error["line_number"],
+                "column": error["column_number"],
+                "message": error["text"],
+                "name": error["code"], "source": "flake8"
+            }
```

Comment:
The logic here duplicates the `format_linter_error` function. Consider using the `format_linter_error` function to format each error, which would make the code more maintainable and reduce duplication.